services/site/src/pages/about.tsx

```diff
@@ -0,0 +1,210 @@
+        <meta name="twitter:card" content="summary_large_image" />
+        <meta name="twitter:site" content="@a11yphant" />
+        <meta name="twitter:title" content="a11yphant" />
+        <meta name="twitter:description" content="a11yphant is an interactive online course for web accessibility." />
+        <meta name="twitter:image" content="https://a11yphant.com/images/mockups-social-media.jpg" />
+        {/* <!-- General Meta Tags --> */}
+        <meta name="theme-color" content="#121212" media="(prefers-color-scheme: dark)" />
+        <meta name="theme-color" content="#FFFFFF" media="(prefers-color-scheme: light)" />
+      </Head>
+      <Navigation />
+      <main className={clsx("h-full box-border max-w-screen-3xl mx-auto")}>
+        <div className={clsx("mx-8 py-8 h-main max-w-screen-3xl", "sm:mx-12", "lg:mt-12 lg:mx-24")}>
+          <h1 className={clsx("mb-8", "h2", "md:h1")}>About</h1>
+
+          <div className="mt-4 flex items-center max-w-80ch">
+            <A11yphantLogoWithoutText className={clsx("w-28 mr-4 hidden", "sm:block")} />
+            <div>
+              <p>
+                <span aria-label="Allyphant">a11yphant</span> <span aria-hidden="true">(pronounced [ˈɛlifənt])</span> teaches developers the basics of
+                web accessibility. Learn step by step by completing short, interactive coding challenges and quizzes.
+              </p>
+            </div>
+          </div>
+
+          <p className="my-8">
+            In an ideal world, all websites are designed and developed so everyone can use them. Sadly, many websites do not comply with web
+            accessibility guidelines. Therefore, especially people with impairments and disabilities are barred from using them. One of the main
+            problems is that many developers don't know enough about web accessibility. <span aria-label="Allyphant">a11yphant</span> wants to improve
+            this situation by giving them a tool for learning accessibility in a fun, interactive way.
+          </p>
+
+          <div className="max-w-80ch">
+            {prefersReducedMotion ? (
+              <Image
+                alt="The a11yphant website where users can complete interactive challenges on web accessibility."
+                src={a11yphantStatic}
+                quality={80}
+                placeholder="blur"
+                sizes="80ch"
+                priority
+              />
+            ) : (
+              <video autoPlay muted loop>
+                <source src="/videos/a11yphant-showcase.mp4" />
+              </video>
+            )}
+          </div>
+
+          <section className="my-16">
+            <h2 className={clsx("mb-8", "h4", "md:h3")}>
+              But what is <span aria-label="Allyphant">a11yphant</span>?
+            </h2>
+
+            <p>
+              <span aria-label="Allyphant">a11yphant</span> is an interactive online course for web accessibility. In this course, you will revisit
```

Comment:
Please replace ` <span aria-label="Allyphant">a11yphant</span>` in the whole project with:

`<span className="sr-only">allyphant</span><span aria-hidden="true">a11yphant</span>`